src/app/store/effects/router-effects.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { Title } from '@angular/platform-browser';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { RouterNavigationAction, ROUTER_NAVIGATION } from '@ngrx/router-store';
+import { TranslateService } from '@ngx-translate/core';
+import { tap } from 'rxjs/operators';
+import { CustomRouterState } from 'app/store/serializers/custom-router-serializer';
+
+@Injectable()
+export class RouterEffects {
+  constructor(private actions$: Actions, private titleService: Title, private translateService: TranslateService) {}
+
+  navigate$ = createEffect(
+    () =>
+      this.actions$.pipe(
+        ofType(ROUTER_NAVIGATION),
+        tap((data: RouterNavigationAction<CustomRouterState>) => {
+          this.titleService.setTitle(this.translateService.instant(data.payload.routerState.title) + ' - ' + window.location.hostname);
```

Comment:
Translating is good, but then we need translation markers on all route titles :)